tests/init.rs

```diff
@@ -0,0 +1,43 @@
+extern crate mdbook;
+extern crate tempdir;
+
+use tempdir::TempDir;
+use mdbook::MDBook;
+
+
+#[test]
+fn run_mdbook_init() {
```

Comment:
Good point. `run_mdbook_init` is a pretty useless name. I'll rename it to something longer like `base_mdbook_init_should_create_default_content`.